### DISCUSSION TITLE
How to load pre-trained weights before starting the run?

### DISCUSSION THREAD

**Discussion by ajulyav** [NVIDIA/NVFlare#1973] [Q&A] [answered]
Hello,

I am not sure I got the idea how can I start with the pre-trained model. I have .pt file which I sent to all clients.
As well, I have added `"source_ckpt_file_full_name": "/media/Experiment/NVFlare/workspace/init_model/init_global_model.pt"`
to config_fed_server.json.

I am using the template of Prostate project. Should I add the lines to `supervised_learner.py`?
I appreciate your help in advance or any tutorial in future.

Thank you!

**Comment by holgerroth** [comment, marked as answer]
The server will be sending the initial weights for each FL round. If your persistor in config_fed_server.json loads the checkpoint, it means those weights will be sent to each client in the first round. You don't need to modify supervised_learner.py. CC @ZiyueXu77
